src/http-gateways/trustless-gateway.md

```diff
@@ -301,6 +301,26 @@ of their presence in the DAG or the value assigned to the "dups" parameter, as
 the raw data is already present in the parent block that links to the identity
 CID.
 
+## CAR `skip-leaves` (content type parameter)
+
+The `skip-leaves` parameter specifies whether blocks with the multicodec `raw`
+`0x55` must be sent.
+
+It accepts two values:
+- `y`: Blocks with `raw` multicodec MUST NOT be sent.
+- `n`, or unspecified: Blocks with `raw` multicodec MUST be sent.
+
+A gateway MUST NOT assume this field is `y` if unspecified.
+When not specified it always MUST be understood as `n`.
+
+:::note Notes for implementers
+
+A request which is rooted at a `raw` block and has `skip-leaves=y` does not
+make sense and SHOULD NOT be sent by clients, it is fair for servers to
+error in this situation.
+
+:::
```

Comment:
Something 4XX something.
My lazy side tell me that I don't want to do this because then I save on code and I can return an empty car in boxo gateway instead of implementing one more edge case.
I can do it if it feels important to someone but given sending a 200 and returning an empty car is fine outcome I'm not sure we need to do this.

Reply:
I'm not OK with empty CARs for the error case, and I don't particularly like that boxo/gateway is already doing this for other error cases and would rather not embed it as a standard practice here. `415` please.

Reply:
IMO it's different from the other cases in boxo/gateway. If you request a raw block with skip-leaves, and empty car is a correct answer. It is the complete dag minues all the raw blocks, which happen to be an empty set but that is the client's problem and there is nothing the gateway can do about it. This is not a transient error either.

Reply:
I've updated this section in f96a92a5262fba7c9f788192aaea3f37ab0d8f06 and made HTTP 400 (Bad Request) a MUST behavior that we will also enforce via conformance test. 


(error is not 415 as we moved `skip-raw-blocks` from content type to URL query params – see "Alternatives" / "Why not CAR content type parameter ?" section  in IPIP).